llama.cpp

```diff
@@ -664,12 +668,26 @@ static std::map<llm_arch, std::map<llm_tensor, std::string>> LLM_TENSOR_NAMES =
             { LLM_TENSOR_ATTN_OUT,        "blk.%d.attn_output" },
             { LLM_TENSOR_ATTN_ROT_EMBD,   "blk.%d.attn_rot_embd" },
             { LLM_TENSOR_FFN_NORM,        "blk.%d.ffn_norm" },
+            { LLM_TENSOR_FFN_DOWN,        "blk.%d.ffn_down" },
+        },
```

Comment:
Some of Orion's tensors have been deleted - `LLM_TENSOR_FFN_GATE` and `LLM_TENSOR_FFN_UP`. Please restore:

https://github.com/ggerganov/llama.cpp/blob/1cfb5372cf5707c8ec6dde7c874f4a44a6c4c915/llama.cpp#L654-L670

Reply:
Sorry, I was careless when rabase, it has been fixed.